book/content/reproducibility/04/barriers.md

```diff
@@ -34,7 +34,10 @@ This chapter outlines some of those barriers, and a couple of suggestions to get
 
 ## Barrier 6
```

Comment:
This formats the barrier itself as the heading (rather than having the heading say "Barrier 2" and then reading down to know what it is. I think it will make the chapter easier to scan if someone's quickly reading through.
```suggestion
## Takes Time
```